src/test/java/SalesforceModules/OppurtunitiesUtil.java

```diff
@@ -261,71 +167,35 @@ public static void verifyDocumentGenerated(String logStep){
         }
     }
 
-    /**
-     * Navigate to account
-     *
-     *
-     * @param logStep the log
-     */
     public static void goToAccount(String logStep){
         KeywordUtil.waitForVisible(OppurtunitiesObject.getBackToAccount);
         KeywordUtil.click(OppurtunitiesObject.getBackToAccount,logStep);
     }
 
-    /**
-     * mark stage to proposal and navigating back to account
-     *
-     */
-
-    public static void markStage() throws InterruptedException {
+    public static void markStage(){
         openOpportunity("Created opportunity opened");
         selectStageProposalQuote("proposal stage selected");
         setMarkAsStage("stage marked");
         goToAccount("Navigated back to account");
     }
 
-    /**
-     * selecting closed won after marking opportunity as proposal
-     *
-     *
-     * @param logStep the name
-     */
     public static void selectClosedStatus(String logStep){
-        //Clicking on closed status
         KeywordUtil.waitForClickable(OppurtunitiesObject.closedStatus);
-        WebElement element1 = KeywordUtil.getDriver().findElement(OppurtunitiesObject.closedStatus);
-        JavascriptExecutor executor = (JavascriptExecutor) KeywordUtil.getDriver();
-        executor.executeScript("arguments[0].click();", element1);
-
-        //clicking on mark stage button
+        KeywordUtil.click(OppurtunitiesObject.closedStatus,logStep);
         KeywordUtil.waitForClickable(OppurtunitiesObject.selectClosedStage);
-        WebElement element2 = KeywordUtil.getDriver().findElement(OppurtunitiesObject.selectClosedStage);
-        executor.executeScript("arguments[0].click();", element2);
+        KeywordUtil.click(OppurtunitiesObject.selectClosedStage,logStep);
 
-        //select closed won value from dropdown
         KeywordUtil.waitForVisible(OppurtunitiesObject.selectClosedStageDropdown);
-        KeywordUtil.selectByValue(OppurtunitiesObject.selectClosedStageDropdown, "Closed Won", "Closed won selected from dropdown");
+        Select dropdown = new Select(KeywordUtil.getDriver().findElement(OppurtunitiesObject.selectClosedStageDropdown));
+        dropdown.selectByValue("Closed Won");
 
-        //save option
-        try {
-            KeywordUtil.waitForVisible(OppurtunitiesObject.saveClosedWon);
-            KeywordUtil.click(OppurtunitiesObject.saveClosedWon, logStep);
-        }catch(Exception e){
-            KeywordUtil.getDriver().navigate().refresh();
-            KeywordUtil.waitForVisible(OppurtunitiesObject.saveClosedWon);
-            KeywordUtil.click(OppurtunitiesObject.saveClosedWon, logStep);
-        }
+        KeywordUtil.waitForVisible(OppurtunitiesObject.saveClosedWon);
+        KeywordUtil.click(OppurtunitiesObject.saveClosedWon,logStep);
     }
 
-    /**
-     * Enter createOpportunity
-     *
-     *
-     * @param name the name
-     * @param amount the amount
-     * @param stage the stage
```

Comment:
Please take check this file, you have removed almost all methods description comments, review your code before raising the PR